views/partials/header.ejs

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Cluckr</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="/stylesheets/main.css" rel="stylesheet">
+        <link href="/stylesheets/bootstrap.min.css" rel="stylesheet">
+        <script src="/scripts/jquery-3.3.1.slim.min.js"></script>
+        <script src="/scripts/popper.min.js"></script>
+        <script src="/scripts/bootstrap.min.js"></script>
+    </head>
+    <body>
+    <header>
+    <nav class="navbar navbar-expand navbar-dark bg-success">
```

Comment:
<img width="1361" alt="screen shot 2018-12-04 at 10 59 32" src="https://user-images.githubusercontent.com/25013604/49434138-c8827680-f7b3-11e8-8548-d91210b9aa3e.png">

Make sure this is read carefully. It explains where each link in the navbar is supposed to go. As well as displaying the username, etc.